src/commonComponents/button/Button.js

```diff
@@ -0,0 +1,33 @@
+// prop-types
+import Proptypes from "prop-types";
+
+// css
+import "./button.css";
+
+export const TYPES = {
+  PRIMARY: "primary",
+  SECONDARY: "secondary",
+};
+
+const Button = (props) => {
+  const { type = TYPES.PRIMARY, label, className, ...extra } = props;
+  return (
+    <button {...extra} className={`btn ${type} ${className}`}>
+      {label}
+    </button>
+  );
+};
+
+Button.defaultProps = {
+  label: "Submit",
+  className: "",
+  type: "",
```

Comment:
you can give default value for type here, also be careful while using default value here "" can not be default value since it should be on of the constants you mentioned